app/presenters/contacts_finder_presenter.rb

```diff
@@ -0,0 +1,101 @@
+class ContactsFinderPresenter
+
+  def initialize(contact_group)
+    @contact_group = contact_group
+    @organisation = contact_group.organisation
```

Comment:
Yeah, I think it makes it cleaner.  There's not really a performance penalty either as we've already loaded all the contact_groups for the `contact_groups_to_facet` method.